drivers/led_strip/Kconfig.apa102

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2018 Google LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig APA102_STRIP
+	bool "APA102 SPI LED strip driver"
+	depends on SPI
+	depends on !SPI_LEGACY_API
+	select LED_STRIP_RGB_SCRATCH
```

Comment:
Hurray! I added this config option for the APA102 in particular, glad to see a driver for it.